core/modules/layout/layout.admin.inc

```diff
@@ -1285,10 +1285,11 @@ function layout_block_configure_form($form, &$form_state, Layout $layout, Block
   $template_info = layout_get_layout_template_info($layout->layout_template);
   $form['region'] = array(
     '#type' => 'select',
+    '#prefix' => '<div class="element-invisible">',
```

Comment:
Is this still relevant?

I can "unhide" the select list by removing the class, but selecting another region doesn't seem to do anything. After saving the block with another region selected, the block doesn't get moved.

Reply:
Agreed, restored to original code. We can work on adding a method to move blocks via select lists some other issue.